lib/include/jxl/cms_interface.h

```diff
@@ -64,11 +64,20 @@ typedef struct {
  * @param output_profile the colorspace to which JxlCmsInterface::run should
  *        convert the input data.
  * @param intensity_target for colorspaces where luminance is relative
- *        (essentially: not PQ), indicates the luminance of (1, 1, 1). This is
- *        only useful for conversions between a relative luminance colorspace
- *        and an absolute luminance one, in either direction.
- *        @p intensity_target cd/m² in the absolute colorspace should map to and
- *        from (1, 1, 1) in the relative one.
+ *        (essentially: not PQ), indicates the luminance at which (1, 1, 1) will
+ *        be displayed. This is useful for conversions between a relative
+ *        luminance colorspace and an absolute luminance one, in either
```

Comment:
Given that the comment higher says "essentially: not PQ", does that mean that the sentence here then means: for conversion between PQ and another color space? It would be useful to say that in a slightly more direct way

Reply:
That’s fair. I think that was an attempt at “future-proofing” the comment but it might not be worth it if it hurts the present. Is this better?